README.md

```diff
@@ -1 +1,132 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [OPTIONAL] This image is broken on your readme template. This is probably because you haven't added the [murple_logo.png](https://github.com/microverseinc/readme-template/blob/master/murple_logo.png) image to your project. Kindly add it for it to appear